File: HoWo5_6/PhoneCompany/Nokia.java
package HoWo5_6.PhoneCompany;

import HoWo5_6.Phone.Phone;

public class Nokia extends Phone{

    public Nokia(String model, double sizeDisplay, int akb, int price) {
        super(model, sizeDisplay, akb, price);
    }



    @Override
    public void charge() {
        System.out.println("Неужели он разрядился");
    }



    @Override
    public String toString() {
        return "Nokia ["+super.toString()+"]";
    }


}

File: HoWo7/PhoneCompany/Nokia.py
from Phone.Phone import Phone


class Nokia(Phone):

    def __init__(self, model: str, sizeDisplay, akb: int, price: int):
        super().__init__(model, sizeDisplay, akb, price)

    def __str__(self) -> str:
        return f"Nokia [{super().__str__()}]"